app/assets/stylesheets/admin.scss

```diff
@@ -315,6 +315,16 @@ $table-header:   #ecf1f6;
     .proposal-new {
       padding-top: 0;
     }
+
+    form {
```

Comment:
I've just realized this also happens in public sections such as "My account". Maybe we can use a more generic solution, and apply it just to check boxes :thinking:.

```diff
diff --git a/app/assets/stylesheets/layout.scss b/app/assets/stylesheets/layout.scss
index d26617846..b4eecfd5f 100644
--- a/app/assets/stylesheets/layout.scss
+++ b/app/assets/stylesheets/layout.scss
@@ -1063,6 +1063,10 @@ form {
     line-height: $line-height;
   }
 
+  .checkbox-label {
+    display: table;
+  }
+
   fieldset legend {
     font-weight: bold;
   }
diff --git a/lib/consul_form_builder.rb b/lib/consul_form_builder.rb
index 5eb292079..9d2453458 100644
--- a/lib/consul_form_builder.rb
+++ b/lib/consul_form_builder.rb
@@ -25,7 +25,11 @@ class ConsulFormBuilder < FoundationRailsHelper::FormBuilder
     else
       label = tag.span sanitize(label_text(attribute, options[:label])), class: "checkbox"
 
-      super(attribute, options.merge(label: label, label_options: label_options_for(options)))
+      # TODO: add test; note adding options[:label_options][:class] would overwrite it; is it OK like this?
+      super(attribute, options.merge(
+        label: label,
+        label_options: { class: "checkbox-label" }.merge(label_options_for(options))
+      ))
     end
   end
```